docs/conf.py

```diff
@@ -104,3 +104,8 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
+
+
+def setup(app):
+    """Add custom CSS defined in _static/custom.css."""
+    app.add_css_file('custom.css')
```

Comment:
This injects some custom CSS into the Sphinx generated site, allowing the text in tables to wrap from one line to the next, so all the long lines don't require horizontal scrolling in the database table description columns.